Client/src/Components/DataCellAnalysis/DataCellAnalysisView.tsx

```diff
@@ -49,7 +49,7 @@ export const DataCellAnalysisView = (props: IProps) => {
           <Typography>Data Cell Analysis Upload Page</Typography>
           <FileUploadForm
             onSubmit={handleSubmit}
-            validateFile={validatePNG}
+            isValidFile={validatePNG}
```

Comment:
I thought we were only going to have 1 page for all upload types? Since the backend accepts any file type together

Reply:
i think it should be refactored when the button is also on the new dataset page, and we have a proper png parser page, in the mean time this is jusat extra validation, if its unnecessary we can remove it later, i say it stays for now, since we are just parsing jsons at the moment